contracts/force/force.cpp

```diff
@@ -41,6 +41,20 @@ void force::editcampaign(uint32_t campaign_id, vaccount::vaddress owner, content
                     c.reward = reward;
                   });
 }
+
+void force::delcampaign(uint32_t campaign_id, vaccount::vaddress owner, content content,vaccount::sig sig) {
```

Comment:
Let's rename this to `rmcampaign`
As the name for creation is `mkcampaign`, this is more in line with expectations

Reply:
Content is not necessary for deleting a campaign (the parameter is not being used in the function)